### DISCUSSION TITLE
Django app for registering participants for webinars etc.

### DISCUSSION THREAD

**Discussion by MacLake** [pretix/pretix#3643] [Show and tell]
In 2020 I wrote a Django app that can register participants for a webinar on WebinarJam or Zoom and for a virtual exhibition on ExpoIP. It actually doesn’t contain much code specific to our institute, mainly just the templates for the notification mails, so that I could easily release it as open source if there’s any interest.

The app is triggered by a Pretix web hook, then fetches the participant information from Pretix’ REST API and then immediately registers the user for webinar and if it was successful it returns a 200 OK status in order to let Pretix know. Otherwise it returns with an error, so Pretix will try again to apply the web hook. When registering for a webinar, the provider returns user specific session links that then sent by e-mail to the participants.

For the configuration of a specific webinar this app can fetch information from both Pretix and WebinarJam. You just need to provide the Pretix event code and the WebinarJam webinar id. For the confirmation mail you can define different brandings for different types of webinars.

Currently we still use it regularly for registering users for WebinarJam for paid webinars. We had used it for Zoom and ExpoIP.  But with a couple of lines of code you could register for any other service that offers an API for it.

It’s just written for our purposes, but if you are interested, I can remove the template text that is specific to our institute and upload the code on Github and make a pip package.

Greetings from Darmstadt,
Jens-Erik

**Comment by raphaelm** [comment]
Sounds like a very cool webhook use case! Nice!
